src/main/java/frc/robot/RobotContainer.java

```diff
@@ -195,6 +203,17 @@ private void configureButtonBindings() {
                 .ignoringDisable(true));
   }
 
+  public void configureAlgae() {
+    // algaeCmd = new AlgaeCmd(algaeSubsystem);
+    controller.rightTrigger().onTrue(new InstantCommand(() -> algaeSubsystem.setLiftPosition(0)));// change the zero
+    controller.rightBumper().onTrue(new InstantCommand(() -> algaeSubsystem.setIntakeSpeed(0.5)));
+    controller.rightBumper().onFalse(new InstantCommand(() -> algaeSubsystem.setIntakeSpeed(0)));
+    controller.leftBumper().onTrue(new InstantCommand(() -> algaeSubsystem.setIntakeSpeed(-0.5)));
+    controller.leftBumper().onFalse(new InstantCommand(() -> algaeSubsystem.setIntakeSpeed(0)));
+    algaeSubsystem.setDefaultCommand(algaeCmd);
```

Comment:
I think there is a way to have a command that is like "is on, set this value, is off, turn to 0.